web/beacon-app/src/features/organizations/components/OrganizationDetails/OrganizationDetails.tsx

```diff
@@ -1,14 +1,40 @@
+import { Toast } from '@rotational/beacon-core';
 import { useState } from 'react';
 
 import { BlueBars } from '@/components/icons/blueBars';
 
+import { useFetchOrg } from '../../hooks/useFetchOrgDetail';
 import { DeleteOrg } from '../DeleteOrg';
 
 export default function OrganizationDetails() {
-  const [showButton, setShowButton] = useState(false);
+  const [isOpen, setIsOpen] = useState(false);
 
-  const handleOpen = () => setShowButton(true);
-  const handleClose = () => setShowButton(false);
+  const handleToggleBars = () => {
+    const open = isOpen;
+    setIsOpen(!open);
+  };
+
+  const handleClose = () => setIsOpen(false);
+
+  const { org, hasOrgFailed, isFetchingOrg, error } = useFetchOrg('orgID');
```

Comment:
`useFetchOrg('orgID')` may not be the best way to display this. I'm available to discuss this today, if you'd like.

Reply:
The orgID should be fetched from another hook right ?